### DISCUSSION TITLE
IB target list indent

### DISCUSSION THREAD

**Discussion by Marko64** [maxi-blocks/maxi-blocks#4038] [Q&A]
For a page with only a few containers, the list where you choose the target block is already untidy:
![Screenshot_1](https://user-images.githubusercontent.com/19425503/206153740-9e8882a0-2de4-4f9f-92e8-8de1e6947701.jpg)

Perhaps we should make it indented like this:
![Screenshot_2](https://user-images.githubusercontent.com/19425503/206153910-6c4edd06-e3fe-4c62-bc8f-e5f26903ff7b.jpg)

**Comment by Naaaaiix** [comment]
I think this issue gets further than what we see here. Going to write to have correct context as it affects more than one issue, and IMO we are spending many efforts on patching something it should be simple from the beginning.

As @Marko64 shares here, the way we pick the target on IB can be annoying and confusing. The main idea was to have a component similar to the List View that Gutenberg offers, but its implementation was hard and long, so decided to do a selector as it is now as MVP. At the beginning the list was showing the `uniqueID` of each block, but it was hard for creators to find the correct block as there is no reference to `uniqueID` on the blocks, so they couldn't know which one they were connecting. So, with that problem, we decided to use the `customLabel` of each block to make it easier for creators. Now, we are dealing for long time with this PR #3710 to avoid creators to get confused when duplicating blocks, and we start discussing this concrete possible implementation. 

The root of the question here is: we need to think on a better method to choose a target on IB. Where and how we want to apply our energies? Is this the way we want to do it?
